scalyr_agent/builtin_monitors/windows_event_log_monitor.py

```diff
@@ -724,6 +736,14 @@ def __init__(self, config, logger, channels):
             win32evtlog.EvtRenderContextSystem
         )
 
+        self._placeholder_render = config.get("placeholder_render")
+        if self._placeholder_render:
+            self._dll_cache = Cache(size=10)
```

Comment:
Does it make sense to make those size parameters user configurable?

Reply:
For pure observability purposes, it may make sense to periodically log under INFO (e.g. once a day or every 12 hours) size of those caches + perhaps also keys of the dll_cache.

Reply:
Will do